hazelcast/src/main/java/com/hazelcast/transaction/impl/TransactionContextImpl.java

```diff
@@ -129,6 +132,18 @@ public TransactionalObject getTransactionalObject(String serviceName, String nam
         return obj;
     }
 
+    private boolean requiresBackupLogs(String serviceName) {
```

Comment:
I believe it would be safer to check if `serviceName` is equal to one of the Collection services.
It is fine currently but what if we introduce another `TransactionalService`, say `ReplicatedMap` for example.
Developer can easily miss to update this method.